client/src/components/forms/FormTrackEdit/_FormTrackEdit.scss

```diff
@@ -0,0 +1,41 @@
+.form-track-edit {
+    display: grid;
+    grid-row-gap: 2em;
+    max-width: 30rem;
+    
+    .duration-container {
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+  
+    }
+  
+    .minutes, .seconds {
+      width: 75px;
+    }
+  
+    .duration-label, .duration-colon {
+      font-size: 1.8rem;
+      font-family: inherit;
+      color: $color-primary;
+    }
+  
+    .duration-label {
+      order: 1;
+      margin-right: 5px;
+    }
+  
+    .minutes {
+      order: 2;
+    }
+  
+    .duration-colon {
+      order: 3;
+      width: 10px;
+      text-align: center;
+    }
+  
+    .seconds {
+      order: 4;
```

Comment:
fyi - I do not think the `order` property is necessary since this numbering is following the order that's in the HTML. We can have it here, just something to know in case this comes up on coding interviews (I could see someone asking whether it's necessary).